.circleci/config.yml

```diff
@@ -1129,17 +1180,21 @@ workflows:
             - connector-flow
             - connector-mongodb
       - meta:
+          <<: *ignore_cherry_pick_branches
           requires:
             - connector-catalog
       - server:
+          <<: *ignore_cherry_pick_branches
           requires:
             - integration
             - connector-catalog
             - common
       - s2i:
+          <<: *ignore_cherry_pick_branches
           requires:
             - server
       - upgrade:
+          <<: *ignore_cherry_pick_branches
```

Comment:
I can only imagine that this upgrade job is already using a filter setting and bringing in the template to ignore cherry pick branches screws up the config

Reply:
Good catch, should be fixed, thanks!